runbot/runbot.py

```diff
@@ -1541,10 +1544,20 @@ def build(self, build_id=None, search=None, **post):
         return request.render("runbot.build", context)
 
     @http.route(['/runbot/build/<build_id>/force'], type='http', auth="public", methods=['POST'], csrf=False)
-    def build_force(self, build_id, **post):
-        registry, cr, uid, context = request.registry, request.cr, request.uid, request.context
+    def build_force(self, build_id, search=None, **post):
+        registry, cr, uid = request.registry, request.cr, request.uid
         repo_id = registry['runbot.build']._force(cr, uid, [int(build_id)])
-        return werkzeug.utils.redirect('/runbot/repo/%s' % repo_id)
+        return werkzeug.utils.redirect('/runbot/repo/%s' % repo_id + ('?search=%s' % search if search else ''))
+
+    @http.route(['/runbot/build/<build_id>/kill'], type='http', auth="public", methods=['POST'], csrf=False)
+    def build_kill(self, build_id, search=None, **post):
+        registry, cr = request.registry, request.cr
+        build = registry['runbot.build'].browse(cr, SUPERUSER_ID, build_id)
```

Comment:
shall I adapt `/runbot/build/<build_id>/force`'s route too ? Not really sure why we're not using auth=user there too.

Still, group_user should be able to kill builds, but they do not have the access rights to do so, so SUPERUSER_ID will need to be user, no?